libpdbg/target.c

```diff
@@ -507,7 +507,14 @@ int ocmb_getscom(struct pdbg_target *target, uint64_t addr, uint64_t *val)
 {
 	struct ocmb *ocmb;
 
-	assert(pdbg_target_is_class(target, "ocmb"));
+	assert(pdbg_target_is_class(target, "ocmb") || is_child_of_ody_chip(target));
+
+	/*It is memport or perv under ocmb, 
+		so we need to translate before calling getscom*/
+	if(is_child_of_ody_chip(target))
+	{
+		target = get_class_target_addr(target, "ocmb", &addr);
+	}
```

Comment:
we can add a todo and point to our refactor commit as this code should go away after we refactor